tests/src/test/resources/testng.xml

```diff
@@ -35,16 +35,9 @@
         <classes>
             <class name="org.wso2.micro.gateway.tests.apikey.APIKeyTestCase" />
             <class name="org.wso2.micro.gateway.tests.jwtRevocation.JWTRevocationSupportTestCase" />
-            <class name="org.wso2.micro.gateway.tests.security.AuthenticationFailureTestCase" />
```

Comment:
Is removed test cases covered in somewhere else?

Reply:
Yes, other classes are extending the removed ones. So only extended class name is required in the testng.xml